root.spec

```diff
@@ -146,7 +163,43 @@ done
 export ROOT_INCLUDE_PATH
 
 export ROOTSYS=%i
+
 make INSTALL="$cp" INSTALLDATA="$cp" install
+# Handle debug symbols. We always have a debug subpackage for debug symbols so
+# no point in complicating the standard builds with this.
+
+%if "%{?subpackageDebug:set}" == "set"
+  ELF_DIRS="%i/lib %i/bin"
+  DROP_SYMBOLS_DIRS=""
+
+# optimise the debug symbols, compress them, and split them into separate file
```

Comment:
`scram-project-build.file` contains: `dwz -m .debug/common-symbols.debug -M common-symbols.debug $ELF_BINS || true` it rewrites debug information in binaries and splits of common parts. E.g. you don't need debug information for `std::list` in every single binary. It splits it off into a command debug file and creates a link. This + compressed debug sections lowers debug information size few times.